src/nuget/xunit.console.netcore.nuspec

```diff
@@ -18,26 +18,26 @@
     </description>
     <copyright>Copyright © Microsoft Corporation</copyright>
     <dependencies>
-      <group targetFramework="netstandard1.3">
-        <dependency id="System.Collections" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Collections.Concurrent" version="4.0.12-rc3-24117-00" />
-        <dependency id="System.Console" version="4.0.0-rc3-24117-00" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Globalization" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.IO" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.IO.FileSystem" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.IO.FileSystem.Primitives" version="4.0.1-rc3-24117-00" />
-        <dependency id="System.Linq" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Runtime.Extensions" version="4.1.0-rc3-24117-00" />
-        <dependency id="System.Text.RegularExpressions" version="4.0.12-rc3-24117-00" />
-        <dependency id="System.Threading" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Threading.Tasks" version="4.0.11-rc3-24117-00" />
-        <dependency id="System.Xml.XDocument" version="4.0.11-rc3-24117-00" />
+      <group targetFramework="netstandard1.0">
```

Comment:
I thought this would be netstandard1.0? The xunit.netcore.extensions is the one that requires 1.1.